docs/index.md

```diff
@@ -30,7 +30,17 @@ introduction/templating
 
 ```{toctree}
 ---
-caption: How-To Guides
+caption: Command Line Interface (CLI)
+maxdepth: 1
+hidden: true
+---
+cli/overview
```

Comment:
Here, it could be dropdown to facilitate search and discovery.

Reply:
I will resolve this in the CLI PR.